sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveContext.scala

```diff
@@ -316,7 +316,7 @@ class HiveContext private[hive](
   }
 
   protected[sql] override def parseSql(sql: String): LogicalPlan = {
-    super.parseSql(substitutor.substitute(hiveconf, sql))
+    sqlParser.parsePlan(substitutor.substitute(hiveconf, sql))
```

Comment:
DDLParser is still used in SQLContext. Do we want to completely remove it? Because I already migrate three commands. I think we can test them all together.